### PR TITLE
Publish seed chart

### PR DESCRIPTION
**Pull request by niqdev**

### CODE REVIEW

File: seed/publish.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (mduarte) [inline seed/publish.sh:1]: I would prefer not to have scripts in the middle of the resources. Can we put this script in the root or in a folder just for scripts?

File: seed/requirements.yaml [hunk 2]
```diff
@@ -1,7 +1,5 @@
-# FIXME https://github.com/edgelevel/gitops-k8s/issues/6
 dependencies:
-  - name: argo-cd
-    alias: argocd
-    version: 0.2.2
-    repository: https://argoproj.github.io/argo-helm
+  - name: argocd
+    version: 1.0.0-0
```
Comment (mduarte) [inline seed/requirements.yaml:3]: `-0` ???

Comment (niqdev) [reply seed/requirements.yaml:3]: Please have a look at [this](https://github.com/edgelevel/helm-charts/pull/2#discussion_r287355342)

Comment (conorfennell) [reply seed/requirements.yaml:3]: @niqdev I had a quick look into versioning this morning. Looks like they have an idiom for this already.
`version: A SemVer 2 version (required)` <- this is the chart version
`appVersion: The version of the app that this contains (optional). This needn't be SemVer.` This is the application version i.e. version 1.0.0 of ArgoCD

https://github.com/helm/helm/blob/master/docs/charts.md#the-chartyaml-file

File: seed/publish.sh (hunk 1 above)
Comment (niqdev) [reply seed/publish.sh:1]: I agree, but I would do it as part of a different PR in which we integrate the publish with Travis

File: seed/requirements.yaml (hunk 2 above)
Comment (niqdev) [reply seed/requirements.yaml:3]: Interesting [this](https://github.com/helm/helm/blob/master/docs/charts.md#the-appversion-field), but
```
This field is informational, and has no impact on chart version calculations.
```

**Review by conorfennell — APPROVED** [review]

**Comment by niqdev** [comment]
I've added your feedback in the TODOs list in the readme for now

**Pull request merged**